snap/snapcraft.yaml

```diff
@@ -47,9 +50,11 @@ parts:
       - -DAUTO_DETECT_ISA=OFF
       - -DHAVE_SSE=True
       - -DHAVE_AVX=True
-    source: https://github.com/srsran/srsRAN
+    source: https://github.com/srsran/srsRAN_4G
     source-type: git
-    source-tag: release_22_10
+      #source-tag: release_22_10
```

Comment:
Do we need to keep this tag?

Reply:
Yes, the release is not entirely complete upstream, and the new tag does not exist, so this will need to be changed to the release tag once it is made official.